CHANGES/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
By the way, how about adding a more useful file, like README? It'd show up whenever the users open this dir in the GH web UI. So it's nice to have some instructions.
For example, here's what I've added for setuptools: https://github.com/pypa/setuptools/tree/main/changelog.d#adding-change-notes-with-your-prs

(as a bonus, you may `.. include::` it into the sphinx docs as well: https://setuptools.pypa.io/en/latest/development/developer-guide.html#adding-change-notes-with-your-prs)

Reply:
Good idea!